website/src/controllers/api-v3/groups.js

```diff
@@ -616,4 +618,96 @@ api.inviteToGroup = {
   },
 };
 
+/**
+ * @api {post} /groups/:groupId/quests/invite Invite users to a quest
+ * @apiVersion 3.0.0
+ * @apiName InviteToQuest
+ * @apiGroup Group
+ *
+ * @apiParam {string} groupId The group _id (or 'party')
+ *
+ * @apiSuccess {Object} Quest Object
+ */
+api.inviteToQuest = {
```

Comment:
This route was moved to a separate quests.js file. You should remove these lines and add your reject route to it.